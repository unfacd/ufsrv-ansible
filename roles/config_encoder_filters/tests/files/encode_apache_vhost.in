content:
  - sections:
    - name: VirtualHost
      param: "*:80"
      content:
        - options:
          - DocumentRoot: /www/example1
          - ServerName: www.example.com
          - ErrorLog: /var/log/httpd/www.example.com-error_log
          - CustomLog:
            - /var/log/httpd/www.example.com-access_log
            - common
          - "#": Other directives here ...
