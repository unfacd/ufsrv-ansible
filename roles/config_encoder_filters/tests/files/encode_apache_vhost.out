<VirtualHost *:80>
  DocumentRoot /www/example1
  ServerName www.example.com
  ErrorLog /var/log/httpd/www.example.com-error_log
  CustomLog /var/log/httpd/www.example.com-access_log common
  # "Other directives here ..."
</VirtualHost>
